tardis_env3.yml

```diff
@@ -37,6 +37,9 @@ dependencies:
   - matplotlib-base
   - ipywidgets
   - plotly
+  - panel
+  - watchfiles
```

Comment:
What is this used for?

Reply:
Panel recommends this library during installation, so I put it there.
https://panel.holoviz.org/getting_started/installation.html#installing-panel

Reply:
Hmm, seems to be for development purposes. I'm not sure if we want it as part of the standard TARDIS environment. @wkerzendorf  thoughts?